localstack/config.py

```diff
@@ -310,7 +313,7 @@ def populate_configs(service_ports=None):
         port_var_name = 'PORT_%s' % key_upper
         port_number = service_port(key)
         globs[port_var_name] = port_number
-        url = '%s://%s:%s' % (get_protocol(), LOCALSTACK_HOSTNAME, port_number)
+        url = 'http%s://%s:%s' % ('s' if USE_SSL else '', LOCALSTACK_HOSTNAME, port_number)
```

Comment:
Looks like there may be a merge issue with a previous commit - can we remove this change from the PR, or is there a particular reason for it? (Same for the other related changes in this file)

Reply:
this wasn't mine.. I pulled before i pushed.. I assumed it came from that.. I will remove it